pkgs/applications/misc/sweethome3d/darwin/default.nix

```diff
@@ -0,0 +1,58 @@
+{
+  lib,
+  stdenv,
+  fetchzip,
+  fetchurl,
+  makeBinaryWrapper,
+  zulu,
+  ant,
+  darwin,
+
+  pname,
+  version,
+  src,
+  meta,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  inherit
+    pname
+    version
+    src
+    meta
+    ;
+
+  patches = [
+    ./build-xml.patch
+    ./config.patch
+  ];
```

Comment:
We should share the patches for both builds otherwise someone updating on linux might brick those without even knowing.